hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -766,23 +756,20 @@ def _validate_print_blob(value):
         )
         return replacement
 
-    # Values that aren't bool or PrintSettings will be turned into hard errors
-    # by the 'options' check.
+    check_type(bool, value, "print_blob")
+
     return value
 
 
 settings._define_setting(
     "print_blob",
-    default=PrintSettings.INFER,
+    default=False,
     description="""
-Determines whether to print blobs after tests that can be used to reproduce
+If set to True, Hypothesis will print information after failing tests that can be used to reproduce
 failures.
```

Comment:
Instead of "that can be used to reproduce failures.", which suggests that this is the *only* way to reproduce failures, I'd say "that can be used with the `@reproduce_failure()` decorator." and ensure that's hyperlinked.